Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -963,8 +988,82 @@ def get_mapping_fields_command(service):
     demisto.results(types_map)
 
 
+def get_last_update_in_splunk_time(last_update):
+    """ Transforms the time to the corresponding time on the Splunk server
+
+    Args:
+        last_update: The time to be transformed
+
+    Returns: The corresponding time on the Splunk server
+
+    """
+    last_update_utc_datetime = dateparser.parse(last_update, settings={'TIMEZONE': 'UTC'})
+    dem_params = demisto.params()
+    splunk_timezone = dem_params.get('timezone')
+    if not splunk_timezone:
+        raise Exception('Cannot mirror incidents when timezone is not configured. Please enter the '
```

Comment:
where does this exception come up? how will it look in the logs?

Reply:
need to check this

Reply:
```
2021-02-21 17:22:48.1484 warning unable to execute update mirroring [error 'Script failed to run:
Error: [Traceback (most recent call last):

   File "<string>", line 1371, in <module>
   File "<string>", line 1361, in main
   File "<string>", line 1163, in get_remote_data_command
   File "<string>", line 1142, in get_last_update_in_splunk_time
 Exception: Cannot mirror incidents when timezone is not configured. Please enter the timezone of the Splunk server being used in the integration configuration.
] (2604) (2603)' id '4824' instance 'SplunkPy_instance_1'] (source: /Users/gfreund/dev/go/src/github.com/demisto/server/services/mirroring.go:428)
```